core-bundle/src/Resources/contao/modules/ModuleArticle.php

```diff
@@ -121,7 +121,7 @@ protected function compile()
 
 		// Add the modification date
 		$this->Template->timestamp = $this->tstamp;
-		$this->Template->date = Date::parse($objPage->datimFormat, $this->tstamp);
+		$this->Template->date = Date::parse($objPage ? $objPage->datimFormat : null, $this->tstamp);
```

Comment:
I don't think it's valid to not have a an `$objPage` in this case. Dunno why it was missing though.

Reply:
I suspect this has the same root cause as what’s discussed above for `Controller.php`. The respective log messages appear together.

Reply:
As discussed in Mumble on April 8th, we have to revert this change and make sure to set `$objPage` here: https://github.com/contao/contao/blob/13ca9de98fdef79992a094d962ae9297138a8b24/news-bundle/src/Resources/contao/classes/News.php#L157

Reply:
Done: https://github.com/contao/contao/pull/2946